http-api/src/v1/peer.rs

```diff
@@ -0,0 +1,23 @@
+use crate::Context;
+
+use axum::{response::IntoResponse, routing::get, Extension, Json, Router};
+use librad::PeerId;
+use serde_json::json;
+
+pub fn router(ctx: Context) -> Router {
+    let peer_id = ctx.peer_id;
+
+    Router::new()
+        .route("/peer", get(peer_handler))
+        .layer(Extension(peer_id))
+}
+
+/// Return the peer id for the node identity.
+/// `GET /peer`
```

Comment:
Currently some routes are commented with prefix e.g. `GET /v1/peer` and some are not. I'll rewrite all with no prefix but lmk if you want otherwise. (Full path should be obvious, by looking at the file path)

Reply:
Also [project routes](https://github.com/radicle-dev/radicle-client-services/blob/29416ea9dc207074b4d9c84648fc648f46ce0bf2/http-api/src/lib.rs#L441) lack the `/project` prefix which I'll add.